src/constants/statusCodes.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const BAD_REQUEST = 400;
+const NOT_FOUND = 404;
+const OK = 200;
+const CREATED = 201;
+const UNDERSTOOD = 204;
```

Comment:
Tip* -  you can create object ( in ts - enum) - STATUS_CODES or something like that)